lib/git-fastclone.rb

```diff
@@ -353,12 +353,13 @@ def with_git_mirror(url)
           yield dir
         rescue Terrapin::ExitStatusError => e
           error_strings = [
-            'missing blob object',
-            'remote did not send all necessary objects',
-            /packed object [a-z0-9]+ \(stored in .*?\) is corrupt/,
-            /pack has \d+ unresolved deltas/
+            'fatal: missing blob object',
+            'fatal: remote did not send all necessary objects',
+            /fatal: packed object [a-z0-9]+ \(stored in .*?\) is corrupt/,
+            /fatal: pack has \d+ unresolved deltas/,
+            'error: unable to read sha1 file of '
```

Comment:
This works, although I think you also could have just added `unable to checkout working tree` to the list of error_strings. THis is a fatal message that was emitted when we saw this problem in CI:

```
13:08:37  error: unable to read sha1 file of sqiosbuild/lib/action/action.rb (6113b739af82d8b07731de8a58d6e233301f80ab)
13:08:37  fatal: unable to checkout working tree
13:08:37  warning: Clone succeeded, but checkout failed.
13:08:37  You can inspect what was checked out with 'git status'
13:08:37  and retry with 'git restore --source=HEAD :/'
13:08:37  
```

Reply:
I thought about that, but the message "unable to checkout working tree" seemed to ambiguous and broad, so I thought I'd add the more specific message.